src/main/java/ch/wisv/payments/admin/products/ProductServiceImpl.java

```diff
@@ -130,13 +142,32 @@ public boolean isProductAvailable(Integer productId) {
     }
 
     @Override
-    public void deleteProduct(int productId) {
-        List<Order> orders = orderService.getOrdersByProductId(productId);
+    public ProductGroup getProductGroupById(Integer productGroupId) {
+        return productGroupRepository.findOne(productGroupId);
+    }
 
-        if (orders.size() > 0) {
-            throw new ProductInUseException("Products are already ordered");
+    @Override
+    public void editProductGroup(ProductGroupRequest productGroupRequest) {
+        if (productGroupRequest.getId() != 0) {
+            Committee committee = committeeRepository.findOne(productGroupRequest.getCommitteeId());
```

Comment:
Let's take the `committeeRepository` out of this Service. Use the `CommitteeService` instead